zio-http/src/main/scala/zio/http/netty/Versions.scala

```diff
@@ -0,0 +1,21 @@
+package zio.http.netty
+
+import io.netty.handler.codec.http.HttpVersion
+import zio._
+import zio.http.model.Version
+
+object Versions {
+  import Version._
+  def make(version: HttpVersion)(implicit unsafe: Unsafe, trace: Trace): Version =
```

Comment:
`make` is good for a constructor in the companion object of a data type, but for a utility like this, `convertVersion` is probably better.

Reply:
Maybe `convertNettyToZIO` and `convertZIOToNetty`, respectively?

Reply:
`Versions#make(HttpVersion)` removed and all code now inlined into `SeverInboundHandler`.